Pillar_II/esm/README.md

```diff
@@ -348,27 +348,32 @@ may be useful for others with issues:
 
 #### Running from Alien4Cloud
 
-There is an environment initialization script:
-`/gpfs/projects/dese28/eflows4hpc/esm/fesom2/env.sh`
+Firstly, you must have a Alien4Cloud user account and have a vault ID (SSH key) and you have to add it to the `authorized_keys` file on the systems you want to run your workflows.
```

Comment:
s/a Alien/an Alien

Can you also break the long line into something around ~60-200? It's useful for Git diff and reading on terminal/some editors - https://sembr.org/

Reply:
Roger that!
I understand the link breaks. 
What do you mean by "s/a Alien/an Alien"

Reply:
s is commonly used in some utilities to find/replace text. Shorter for suggesting to replace "a Alien" with "an Alien" :slightly_smiling_face:

Reply:
Thanks for clarifying Bruno. 
Do we need a hard line break using <br/> element or Enter would also work?

Reply:
Just a enter should work. I think we have some text that already follows that pattern. You can just try to stay more or less within the same line length/column as the rest of the document.